File: program/convert-fahrenheit-to-celsius/ConvertFahrenheitToCelsius.c
#include <stdio.h>

int main()
{

    float celsius, fahrenheit;

    printf("enter fahrenheit \n");
    scanf("%f", &fahrenheit);

    celsius = (fahrenheit - 32) * 5 / 9;

    printf("celsuis is %f", celsius);

    return 0;
}

File: program/convert-fahrenheit-to-celsius/convert_fahrenheit_to_celsius.hs
main :: IO ()
main = do 
    putStrLn "Enter temperature in Fahrenheit: " 
    input <- getLine
    let fahrenheit = read input :: Float

    let celsius = (fahrenheit - 32) * 5/9
    putStrLn $ "Given Fahrenheit in Celsius: " ++ show(celsius) ++ " °C"